server/src/main/java/org/eclipse/openvsx/entities/ListOfStringConverter.java

```diff
@@ -0,0 +1,31 @@
+/********************************************************************************
+ * Copyright (c) 2020 TypeFox and others
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ ********************************************************************************/
+package org.eclipse.openvsx.entities;
+
+import org.apache.commons.lang3.StringUtils;
```

Comment:
Are we allowed to use this dependency? It is a transitive Spring Boot dependency...

Reply:
I have used Guava elsewhere in the code, since we have that as an explicit dependency.

Reply:
done